argoexec/rockcraft.yaml

```diff
@@ -74,8 +101,11 @@ parts:
 
   non-root-user:
     plugin: nil
-    after: kfam
+    after: [copy-files]
```

Comment:
This can be removed.  The order for this does not matter
```suggestion
```

Reply:
Yes, you are correct.